src/couch/src/couch_changes.erl

```diff
@@ -370,9 +370,7 @@ get_changes_timeout(Args, Callback) ->
         timeout = Timeout,
         feed = ResponseType
     } = Args,
-    DefaultTimeout = list_to_integer(
-        config:get("httpd", "changes_timeout", "60000")
-    ),
+    DefaultTimeout = chttpd_util:get_chttpd_config(changes_timeout, 60000),
```

Comment:
Good idea to convert it to an integer but we'd want to go through some new `get_chttpd_config_integer/3` calls here probably